src/test/java/com/kodilla/ecommercee/dao/ProductDaoTests.java

```diff
@@ -0,0 +1,94 @@
+package com.kodilla.ecommercee.dao;
+
+import com.kodilla.ecommercee.domain.Cart;
+import com.kodilla.ecommercee.domain.Order;
+import com.kodilla.ecommercee.domain.Product;
+import com.kodilla.ecommercee.domain.ProductsGroup;
+import org.assertj.core.api.Assertions;
+import org.junit.Test;
+import org.junit.jupiter.api.DisplayName;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.Assert.*;
+
+@SpringBootTest
+@DisplayName("ProductDao Test Suites")
+public class ProductDaoTests {
+    @Autowired
+    private ProductDao productDao;
+
+    @Test
+    void testFindAll() {
+        //Given
+        ProductsGroup productsGroup = new ProductsGroup();
+        List<Order> orderList = new ArrayList<>();
+        List<Cart> cartList = new ArrayList<>();
+        Product product = new Product(1L, "name", "description",
```

Comment:
a to Id nie powinno się samo generować, a nie być wpisywane z palca?

Reply:
Ogólnie tak jest ustawione ale przy tworzeniu obiektu klasy Product wpisałam z palca bo nie chciałam tworzyć nowego konstruktora.